ui/app/adapters/ldap/role.js

```diff
@@ -72,7 +73,7 @@ export default class LdapRoleAdapter extends NamedPathAdapter {
     }
     // must return an object in this shape for lazyPaginatedQuery to function
     // changing the responsePath or providing the extractLazyPaginatedData serializer method causes normalizeResponse to return data: [undefined]
-    return { data: { keys: roles.sortBy('name') } };
```

Comment:
Ember's `sortBy` method [docs](https://github.com/emberjs/ember.js/blob/v4.12.0/packages/%40ember/array/index.ts#L1425-L1442) uses `toArray()` so I replaced both of our uses with the `sortObjects` util